tests/utils/project_test_utils.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+import (
+	schema "github.com/devfile/api/pkg/apis/workspaces/v1alpha2"
+)
+
+type GenericProject struct {
+	Name    string
+	Project *schema.Project
+}
```

Comment:
pls delete this file and struct, its not being used anywhere

Reply:
Will remove the file. I have a branch waiting on this which implements this fully.